src/components/Table/GridTable.tsx

```diff
@@ -1413,14 +1431,18 @@ const rowLinkRenderFn: (as: RenderAs) => RenderCellFn<any> =
 
 /** Renders a cell that will fire the RowStyle.onClick. */
 const rowClickRenderFn: (as: RenderAs, api: GridTableApi<any>) => RenderCellFn<any> =
-  (as: RenderAs, api: GridTableApi<any>) => (key, css, content, row, rowStyle, classNames: string | undefined) => {
+  (as: RenderAs, api: GridTableApi<any>) =>
+  (key, css, content, row, rowStyle, classNames: string | undefined, onClick) => {
     const Row = as === "table" ? "tr" : "div";
     return (
       <Row
```

Comment:
**request:** This isn't your fault, but it really confused me when reading this. We **aren't** rendering a `Row` here, but instead a `Cell`. This naming and the use of a `tr` tag is incorrect in this situation. Would you mind fixing this and updating it to:
```
const Cell = as === "table" ? "td" : "div";
return (
  <Cell>...</Cell>
);
```
Please and thanks!